WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2562,25 +2771,7 @@ static GSourceFuncs _handlerIntervention =
             if (isWebProcessResponsive && _unresponsiveReplyNum == 0)
                 return;
 
-#ifdef WEBKIT_GLIB_API
-            std::string activeURL(webkit_web_view_get_uri(_view));
```

Comment:
activeURL is still needed I suppose.

Reply:
Yes. activeURL is needed. I added this line back.